core/add_test.go

```diff
@@ -49,20 +48,17 @@ func TestAddDone(t *testing.T) {
 func contextWithStore(f func(ctx context.Context, store Store)) {
 	ctx := context.Background()
 
-	db, err := storm.Open("/tmp/test123.db")
+	store, err := NewBoltStore("/tmp/test123.db")
```

Comment:
really should update these to use random temp files from ioutils. But orthogonal change to this